search_action_domain/models/ir_actions_server.py

```diff
@@ -0,0 +1,40 @@
+from odoo import models
+
+
+class IrActionsServer(models.Model):
+    _inherit = "ir.actions.server"
+
+    def create_as_ir_filter(self):
+        model_data = self.env["ir.model.data"].sudo()
+        for record in self:
+            if not record.state == "code":
+                continue
+            xid = "%s.ir_filters_server_action_%s" % (
+                record.model_id.model.replace(".", "_"),
+                record.id,
+            )
+            if self.env.ref(xid, raise_if_not_found=False):
+                continue
+            created_filter = self.env["ir.filters"].create(
+                {
+                    "name": record.name,
+                    "model_id": record.model_id.model,
+                    "domain": "[]",
+                    "context": "{'search_action_domain': {'%s': [%s]}}"
+                    % (record.model_id.model, record.id),
+                    "active": True,
+                    "user_id": False,
+                    "is_default": False,
```

Comment:
I think this is already the default. If so, remove this and other defaults, so if those are changed and button is re-pressed, those values are not lost

Reply:
That's what I meant. If it doesn't have a default value, then `False` is the default values.

I think we should include only those values that differ from their defaults